api/openapi/v1/core-metadata.yaml

```diff
@@ -2143,11 +2156,14 @@ components:
       title: device
       type: object
       properties:
-        addressable:
-          $ref: '#/components/schemas/device_addressable'
         adminState:
           title: adminState
           type: string
+        autoevents:
+          type: array
+          items:
+            title: autoevent
+            $ref: '#/components/schemas/autoevent'
```

Comment:
Please add a corresponding autoevent schema.

Reply:
...unless I'm misunderstanding something and it can reference a schema inside core-contracts

Reply:
This is referencing the object `autoevent` on line 2119.